### PR TITLE
Created a Status Log page, Linked to 500 Template

### PR DESCRIPTION
**Pull request by harry-7**
For Issue #451 Necessary changes are made

### CODE REVIEW

File: plinth/modules/help/help.py [hunk 1]
```diff
@@ -71,3 +71,14 @@ def manual(request):
         request, 'help_manual.html',
         {'title': _('{box_name} Manual').format(box_name=_(cfg.box_name)),
          'content': content})
+
+def status_logs(request):
+    X = 100
```
Comment (fonfon) [inline plinth/modules/help/help.py]: Variable names should be as descriptive as possible (without being too verbose), I'd propose something like 'nr_lines' instead.
Also, upper case variables are not PEP compliant - please check your code with a validation tool like pylint or flake8.

File: plinth/modules/help/templates/statuslog.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    #
+{% endcomment %}
+
+{% load i18n %}
+
+{% block content %}
+
+<p>
+    {% blocktrans trimmed %}
+        This is the last {{ X }} lines of status log Please attatch with the bug when raising a bug.
```
Comment (fonfon) [inline plinth/modules/help/templates/statuslog.html]: I'd rephrase this a bit ('report a bug', missing point between sentences);
I cannot run the code of this pull request now, so I only look at the text and syntax.

File: plinth/modules/help/help.py [hunk 3]
```diff
@@ -71,3 +71,14 @@ def manual(request):
...
+    file = open(cfg.status_log_file,"r")
```
Comment (fonfon) [inline plinth/modules/help/help.py]: Closing the file is missing

**Comment by fonfon** [comment]
In general this looks good; I can't run the code though (have no freedombox/vm at hands).

File: plinth/modules/help/templates/statuslog.html (hunk 2 above)
Comment (harry-7) [inline plinth/modules/help/templates/statuslog.html]: Report a Bug thing is already there in the 500 template and we are redirecting here. Wont it be sufficient ?

**Comment by harry-7** [comment]
@fonfon I made a commit with necessary changes. Please have a look and suggest any changes

**Comment by fonfon** [comment]
Looks good - but I cannot test/run the code now so it'd be great if somebody else could do that

**Comment by jvalleroy** [comment]
I have rebased and merged these changes. I also made some minor style changes, and added a warning to remove passwords from the log before submitting a bug report.
